pkg/collector/reconcile/configmap.go

```diff
@@ -86,8 +93,10 @@ func desiredConfigMap(_ context.Context, params Params) corev1.ConfigMap {
 
 func desiredTAConfigMap(params Params) (corev1.ConfigMap, error) {
 	name := naming.TAConfigMap(params.Instance)
+	version := strings.Split(params.Instance.Spec.Image, ":")
 	labels := targetallocator.Labels(params.Instance)
 	labels["app.kubernetes.io/name"] = name
+	labels["app.kubernetes.io/version"] = version[len(version)-1]
```

Comment:
do we need to check the array length here as well? and would it make sense to maybe outsource this check to a small function?

Reply:
@frzifus, I've just included the missing part. As we are doing this only twice I guess It could be in that format.